### PR TITLE
Add basic submodules documentation

### PR DESCRIPTION
**Pull request by mbohlool**
We are using submodules to handle dependencies to base repo and we need a reference docs for steps of updating it.

### CODE REVIEW

File: devel/submodules.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+```
+
+# Update submodule
+If you changed kubernetes-client/python-base and want to pull your changes into this repo run this command:
+
+```bash
+git submodule update --remote
+```
+
+and commit changes and send a PR for it.
```
Comment (pokoli) [inline devel/submodules.md]: I will prefer: "Once updated, you should fill a new PR to upload changes for the repository" or something similar. 

What do you think?

Comment (mbohlool) [inline devel/submodules.md]: That's what I mean. You should commit your changes to your clone then send a pull request for your changes.

Comment (pokoli) [inline devel/submodules.md]: Yes, but I will prefer to make it clearer on the docs :)

Comment (mbohlool) [inline devel/submodules.md]: to me `fill` and `upload` doesn't feel right. I prefer `create` and `commit`.

How about:
"Once updated, you should create a new PR to commit changes to the repository".

Comment (pokoli) [inline devel/submodules.md]: Seems a good opition to me.

Comment (mbohlool) [inline devel/submodules.md]: Done. Thanks.

**Comment by pokoli** [comment]
LGTM

**Pull request merged**